download.html

```diff
@@ -20,6 +20,17 @@
 <div class="container">
     <div class="download_main">
 
+    <h2>Installing with go-xcat <sup class="new">New!</sup> </h2>
+
+    To easily install xCAT and handle the dependency package, a new script <a href=http://xcat.org/files/go-xcat>go-xcat</a> is provided.
```

Comment:
Hi @whowutwut 

I do not like the hyperlink you put here. I have a couple of reasons. I listed them below.

* For a shell script with file name extension ``.sh``, the default content type is ``text/plain`` instead of ``application/octet-stream``. This will cause the web browser show the content of the bash script in a tab of the browser instead of download it. Thus, it is a bad user experience.
* Even if we configure our web server to reply a proper content type for this script, the end user has to download this piece of bash script to his/her own workstation, and then upload the script to his/her Linux box somewhere in the server room. This is a multiple-step procedure which involved a lot of manual steps, something like ``scp``. Which is complex.
* Just copy the line ``wget http://xcat.org/files/go-xcat -O - >/tmp/go-xcat`` from browser and paste it into the terminal  is quite simple and straight forward. Thus, I prefer my old school method since it has better user experience.